### PR TITLE
new signup flow draft

### PR DESCRIPTION
**Pull request by jlin99**

### CODE REVIEW

File: src/posts/signupflow.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## So what's new? 
+
+1. Auth
+
+For all my API key haters out there, we recognize our previous API key based solution was somewhat daunting and cumbersome, so we're happy to finally put that aside. For all my API key lovers, don't worry! It's not going anywhere. You'll still be able to use it to interact with our API directly. But, now when you sign in to our frontend, you'll be using an email and password or Google SSO! 
+
+2. Workspace
+
+When you sign up, you'll be able to make your own workplace, invite others, and share all the cool stuff you're cooking up! Or join someone else's workspace and see what they've been creating.
```
Comment (GradySalzman) [inline src/posts/signupflow.md:23]:
```suggestion
Join forces and collaborate with others on your content more seamlessly. You'll now be able to make your own workplace, join others workspaces, and invite others. Once in a workspace your team can submit, see, and edit each others posts! 
```

File: src/posts/signupflow.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+tags: ["announcements"]
+---
+
+If you were up during demon hours last night (for your sake, I hope you weren't), you might have noticed that we were under maintenance. It was all so we can roll out our new platform!
+
+## So what's new? 
+
+1. Auth
+
+For all my API key haters out there, we recognize our previous API key based solution was somewhat daunting and cumbersome, so we're happy to finally put that aside. For all my API key lovers, don't worry! It's not going anywhere. You'll still be able to use it to interact with our API directly. But, now when you sign in to our frontend, you'll be using an email and password or Google SSO!
```
Comment (GradySalzman) [inline src/posts/signupflow.md:19]: just reworded a little
```suggestion
To all the API key sign in haters out there, we hear you loud and clear! We acknowledge that our previous API key-based sign-in process could be a bit daunting and cumbersome. We're thrilled to announce that we're leaving that behind! And for all you API key lovers, don't worry! You can still use it to interact directly with our API. However, when it comes to signing in on our frontend, we're introducing two new options: email and password or Google SSO. So, get ready for a smoother and more convenient sign-in experience that suits your preferences.
```

**Review by GradySalzman — COMMENTED** [review]
Looks good - just small comments

File: src/posts/signupflow.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+author: "Tamás Deme"
+added: "Jun 29 2023"
+tags: ["announcements"]
+---
+
+If you were up during demon hours last night (for your sake, I hope you weren't), you might have noticed that we were under maintenance. This small maintenance window has been the culmination of many many hours as we've been hammering away behind the scenes... and are finally ready to reveal a transformative update that's designed to amplify your journey with us. Hold on to your hats, because we're unlocking a new realm of user experience – proper authentication!
+
+## So what's new? 
+
+Aside from the industry standard security practices now in place to make sure your account is only accessible by you - and only you -, we are pushing forward in 3 major areas.
```
Comment (jlin99) [inline src/posts/signupflow.md:17]:
```suggestion
Aside from the industry standard security practices now in place to make sure your account is only accessible by you –and only you–, we are pushing forward in 3 major areas.
```
just making them em dashes looks better than regular dashes

**Pull request merged**